.github/workflows/build-test-distribute.yaml

```diff
@@ -68,18 +71,35 @@ jobs:
           make clean
       - run: |
           make check
-      - id: sca-project
-        uses: Kong/public-shared-actions/security-actions/sca@0ccacffed804d85da3f938a1b78c12831935f992 # v2.8.0
-        with:
-          dir: .
-          config: .syft.yaml
-          upload-sbom-release-assets: true
-      - id: metadata
+      - name: "Set metadata for downstream jobs"
+        id: metadata
         run: |
           echo "images=$(make images/info/release/json)" >> $GITHUB_OUTPUT
           echo "registry=$(make docker/info/registry)" >> $GITHUB_OUTPUT
           echo "version=$(make build/info/version)" >> $GITHUB_OUTPUT
           echo "distribution_repository=$(make build/info/cloudsmith_repository)" >> $GITHUB_OUTPUT
+      # "make check" puts binaries in ./build/tools (see mk/generate.mk: $(POLICY_GEN)
+      # and $(RESOURCE_GEN)), which are unnecessarily included in the SBOM.
+      # Running the SCA step before "make check" might seem like a solution, but it
+      # generates report files (e.g., sbom.spdx.json, cve-report.json) in the working
+      # directory. These files cause "make check" to fail, as it checks if no files
```

Comment:
what about adding these to `.gitignore`? not sure if that would be better or worse

Reply:
I don't want to do it as if `sca` action would modify names for these files or would add new reports our CI will be broken and if this change will be backported to release branches, it will involve a lot of work to fix this simple issue